cg-spec/editors_draft.html

```diff
@@ -441,7 +441,24 @@ <h2>Structure of this document</h2>
     </section>
 
     <section id="conformance">
-      <div class=issue data-number="3"></div>
+      <p>A system supports RDF-star:</p>
+      <ul>
```

Comment:
It is not entirely clear whether an AND or an OR applies to the items of the list... I could see arguments for both, but I believe OR is more (e.g. converters to/from other syntaxes)

Reply:
I also believe that OR is more useful. To capture this, the line can be extended as follows: "A system supports RDF-star if it has at least one of the following two properties" (and, then, "if" needs to be removed from the two bullet points).

Alternatively, we may split this whole part into two separate statements as follows.

A system supports RDF-star as input if ...

A system supports RDF-star as output if ...